go/vt/key/key_test.go

```diff
@@ -460,7 +685,62 @@ func TestParseShardingSpec(t *testing.T) {
 	}
 }
 
-func TestContains(t *testing.T) {
+func TestKeyRangeComparisons(t *testing.T) {
+	type args struct {
+		a *topodatapb.KeyRange
+		b *topodatapb.KeyRange
+	}
+	type wants struct {
+		wantStartCompare int
+		wantStartEqual   bool
+		wantEndCompare   int
+		wantEndEqual     bool
+		wantCompare      int
+		wantEqual        bool
+	}
+	tests := []struct {
+		name  string
+		args  args
+		wants wants
+	}{
+		{"a and b are both full range", args{stringToKeyRange("-"), stringToKeyRange("-")}, wants{0, true, 0, true, 0, true}},
```

Comment:
This would be much more readable and easier to confirm expected results:
```
{
    name: "a and b are both full range",
    args: args{
       stringToKeyRange("-"), stringToKeyRange("-")
    },
    wants: wants{
       wantStartCompare: 0,
       wantStartEqual: true,
       wantEndCompare: 0,
       wantEndEqual: true,
       wantCompare: 0,
       wantEqual: true,
    }
},
```

Reply:
You just need a good IDE, Matt! But yes, that's fine to me, I'm looking at it always with the fields named by the IDE, so it's no difference to me and makes it easier to see what's going on in other IDEs and on e.g. Github.